edi_edifact_oca/models/edifact.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, models
+
+from odoo.addons.edi_party_data_oca.utils import get_party_data_component
+
+
+class BasePydifact(models.AbstractModel):
```

Comment:
Who should be the consumer of this method?

Reply:
Yes, I think `get_party_data` was not used at all